### PR TITLE
Opensora-PKU: fix fa attention mask

### PR DESCRIPTION
**Pull request by wtomin**
FlashAttention in MindSpore only accepts mask with (0, 1), where 0 means to keep, and 1 means to discard.

Editing the current Opensora-PKU attention mask:
1. comment the code to fill in mask with `-ms.numpy.inf` in `LatteT2V.construct`;
2. Flip the mask using `1 - mask` since FA treats 1 as discard, 0 as retain.
3. Fill in with `-ms.numpy.inf` in vanilla Attention, and leave the mask untouched for FA

Minor changes: 
1. Print `Use FA` in `sample_t2v.py`;
2. Force tensor to fp32 before saving to `npz` in  `sample_text_embed.py`.

### CODE REVIEW

File: examples/opensora_pku/opensora/models/diffusion/latte/modeling_latte.py [hunk 1]
```diff
@@ -1427,24 +1427,27 @@ def construct(
             # assume that mask is expressed as:
             #   (1 = keep,      0 = discard)
             # convert mask into a bias that can be added to attention scores:
-            #       (keep = +0,     discard = -ms.numpy.inf)
+            #       (keep = +0,     discard = -10000.0)
             attention_mask = attention_mask.unsqueeze(1)
-            attention_mask = ops.zeros(attention_mask.shape).masked_fill(~attention_mask, -ms.numpy.inf)
+            # do not fill in -10000.0 until entering MHA
+            # attention_mask = ops.zeros(attention_mask.shape).masked_fill(~attention_mask, -10000.0)
```
[the remark this answers is not in the document]
Comment (wtomin) [reply examples/opensora_pku/opensora/models/diffusion/latte/modeling_latte.py:1433]: Because torch code masked_fill with -10000.0 in this part: [See torch implementation](https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/main/opensora/models/diffusion/latte/modeling_latte.py#L343-L368).

Here I want to use this comment to explain the code difference between ms and torch. Torch will mask_fill before entering MHA, while MS will mask_fill until entering MHA. It's OK to just delete this comment.

The mask flip and broadcast is done inside MHA.

[the remark this answers is not in the document]
Comment (wtomin) [reply examples/opensora_pku/opensora/models/diffusion/latte/modeling_latte.py]: FYI, those misleading comments were deleted.

**Review by geniuspatrick — APPROVED** [review]

**Review by CaitinZhao — APPROVED** [review]
